cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -1,164 +1,92 @@
 # -*- coding: utf-8 -*-
 import fauxfactory
 import pytest
+
+from cfme.cloud.availability_zone import AvailabilityZone
 from cfme.cloud.instance import Instance
-from cfme.cloud.provider import CloudProvider
-from cfme.web_ui import InfoBlock, toolbar, jstimelines
-from cfme.exceptions import ToolbarOptionGreyedOrUnavailable
-from utils import testgen
-from utils import version
+from cfme.cloud.provider.azure import AzureProvider
+from utils import testgen, version
 from utils.appliance.implementations.ui import navigate_to
-from utils.blockers import BZ
 from utils.log import logger
 from utils.wait import wait_for
 
 
-pytest_generate_tests = testgen.generate([CloudProvider], scope="module")
-
-pytestmark = [pytest.mark.tier(2)]
+pytestmark = [pytest.mark.tier(2),
+              pytest.mark.uncollectif(lambda: version.current_version() < '5.7'),
+              pytest.mark.usefixtures("setup_provider_modscope")]
```

Comment:
**REQUIRED:** This can be left to fixture scope as the fixture is clever enough to check via a quick means if the provider is already there.